harbor-core.yaml

```diff
@@ -47,6 +51,74 @@ pipeline:
 
   - uses: strip
 
+subpackages:
+  - name: harbor-jobservice
+    description: harbor jobservice
+    pipeline:
+      - uses: go/build
+        with:
+          packages: ./jobservice
+          output: harbor-jobservice
+          modroot: ./src
+      - runs: |
+          mkdir -p ${{targets.subpkgdir}}/harbor
+          ln -sf /usr/bin/harbor-jobservice ${{targets.destdir}}/harbor/harbor_jobservice
```

Comment:
This means that the subpackage has to depend on the full package right? Is that intentional?

Reply:
No, it doesn't have to depend on the full package. There isn't an implicit dependency created here that brings in `harbor-core` so we're good

Reply:
OH WAIT. I see what you mean. Fix incoming